src/components/ImgSlider/ImgSlider.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import ArrowSvg from '../../helper-components/ArrowSvg';
+import styles from './ImgSlider.module.css';
+
+const ImgSlider = ({ game }) => {
+  const white = '#FFFFFF';
+  const dirtyWhite = '#d1d1d1';
+  const [leftArrowNavFill, setLeftArrowNavFill] = React.useState('#d1d1d1');
```

Comment:
Can you handle this stuff with css `:hover`? Remember all this state is causing re-renders if we can avoid this and use css `:hover` it would simplify the code and improve performance.

Reply:
I thought I had two options here: 
1. use embeded SVGs and then I can use the `filter: fill `property but I thought it results in a pretty verbose code
2. use a react component that renders a SVG from props, in which case I have to use states to change those props

Another option would be to use an <img/> element and use `filter: invert` on it, but that just inverts the colors and I don't have that much control over them.

Maybe there is another option that I miss?